src/phoenix/datasets/dataset.py

```diff
@@ -444,6 +453,23 @@ def _check_embedding_features_schema_field_for_excluded_columns(
     )
 
 
+def _check_embedding_column_names_for_excluded_columns(
+    embedding_column_name_mapping: EmbeddingColumnNames,
+    column_name_to_include: Dict[str, bool],
+    unseen_column_names: Set[str],
+) -> None:
+    """
+    Check embedding features for excluded column names.
+    """
+    included_embedding_features = {}
+    included_embedding_features[""] = deepcopy(embedding_column_name_mapping)
```

Comment:
not sure I follow this syntax of setting it at the "" index. Why not one line on 464?

Reply:
also is the deep copy needed? Somehow things get accidentally preserved by reference?

Reply:
This was a mistake.

Reply:
Resolved in [87c4ed8](https://github.com/Arize-ai/phoenix/pull/609/commits/87c4ed8b0c2db8dcf09f42e72d566d3bbc56dc93)